packages/core/lib/src/core_helpers.dart

```diff
@@ -198,12 +198,12 @@ Uint8List bytesFromDataUri(String dataUri) {
 }
 
 /// Returns [List<T>] if [x] is provided or `null` otherwise.
-Iterable<T> listOrNull<T>(T x) => x == null ? null : [x];
+List<T>? listOrNull<T>(T? x) => x == null ? null : [x];
```

Comment:
Why `Iterable` -> `List`?

Reply:
Oops, don't know why... I will revert it.